config/sidekiq.yml

```diff
@@ -18,10 +18,13 @@
       cron: '0 1 * * *' # Runs once per day (at 1 am)
       class: Updater::ExistingCompanyJobsJob
       queue: updates
-    devitjobs_updater:
+      enabled: <%= Rails.env.production? %> # production only as it interferes with end-to-end testing
+    devitjobs_updater: # off temporarily as interferes with end-to-end testing
       cron: '0 2 * * *' # Runs once per day (at 2 am)
       class: Importer::Api::DevitJobsJob
       queue: updates
+      enabled: <%= Rails.env.production? %> # production only as it interferes with end-to-end testing
```

Comment:
Similarly - why?

Reply:
FormFiller runs as a background job. Can't run if there's three hundred background jobs already in the queue. Also it's just annoying in development to have the database constantly being filled with jobs I didn't seed.